src/msw/webview_ie.cpp

```diff
@@ -853,25 +861,225 @@ wxString wxWebViewIE::GetPageText() const
     }
 }
 
-void wxWebViewIE::RunScript(const wxString& javascript)
+wxWebViewIEEmulationLevel wxWebViewIE::GetEmulationLevel()
+{
+    wxRegKey key(wxRegKey::HKCU, _T(REGISTRY_IE_PATH));
+    if ( key.Exists() )
+    {
+        long val = 0;
+
+        if ( !key.QueryValue(wxGetFullModuleName().AfterLast('\\'), &val) )
+        {
+	  wxLogWarning(_("Can not get level value of registry key %s"), REGISTRY_IE_PATH);
+            return wxWEBVIEWIE_EMULATION_LEVEL_EMPTY;
+        }
+
+        switch ( val )
+        {
+            case wxWEBVIEWIE_EMULATION_LEVEL_11:
+                return wxWEBVIEWIE_EMULATION_LEVEL_11;
+            case wxWEBVIEWIE_EMULATION_LEVEL_11_DEFAULT:
+                return wxWEBVIEWIE_EMULATION_LEVEL_11_DEFAULT;
+            case wxWEBVIEWIE_EMULATION_LEVEL_10:
+                return wxWEBVIEWIE_EMULATION_LEVEL_10;
+            case wxWEBVIEWIE_EMULATION_LEVEL_10_DEFAULT:
+                return wxWEBVIEWIE_EMULATION_LEVEL_10_DEFAULT;
+            case wxWEBVIEWIE_EMULATION_LEVEL_9:
+                return wxWEBVIEWIE_EMULATION_LEVEL_9;
+            case wxWEBVIEWIE_EMULATION_LEVEL_9_DEFAULT:
+                return wxWEBVIEWIE_EMULATION_LEVEL_9_DEFAULT;
+            case wxWEBVIEWIE_EMULATION_LEVEL_8:
+                return wxWEBVIEWIE_EMULATION_LEVEL_8;
+            case wxWEBVIEWIE_EMULATION_LEVEL_8_DEFAULT:
+                return wxWEBVIEWIE_EMULATION_LEVEL_8_DEFAULT;
+            case wxWEBVIEWIE_EMULATION_LEVEL_7_DEFAULT:
+                return wxWEBVIEWIE_EMULATION_LEVEL_7_DEFAULT;
+            default:
+                return wxWEBVIEWIE_EMULATION_LEVEL_EMPTY;
+        }
+    }
+    wxLogWarning(_("Registry %s does not exist"), REGISTRY_IE_PATH);
+    return wxWEBVIEWIE_EMULATION_LEVEL_EMPTY;
+}
+
+bool wxWebViewIE::SetEmulationLevel(wxWebViewIEEmulationLevel level)
+{
+    wxRegKey key(wxRegKey::HKCU, _T(REGISTRY_IE_PATH));
```

Comment:
Perhaps because regular programs are usually not run with Administrator privileges and therefore cannot write to HKLM? Writing the value to HKLM would be the installer's job...